R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
I understand that, but as pointed out https://github.com/apache/spark/pull/16720#issuecomment-275832013, some tests don't need SparkSession, and some tests will create/stop one as needed, and to have a function that does that all just mean more complicity?